packages/protocol/scripts/truffle/make-release.ts

```diff
@@ -72,6 +73,22 @@ class ContractAddresses {
 
 const REGISTRY_ADDRESS = '0x000000000000000000000000000000000000ce10'
 
+const isProxiedContract = async (contractName: string) => {
+  if (contractName.endsWith('Proxy')) {
+    return false
+  }
+
+  try {
+    await artifacts.require(`${contractName}Proxy`)
```

Comment:
I don't think you need to `await` on `artifacts.require`, so this whole function doesn't need to be async.